test/export/test_experimental.py

```diff
@@ -134,6 +134,7 @@ def forward(self, x):
         exported_module = _convert_ts_to_export_experimental(
             traced_module_by_torchscript, inps
         )
+        print(exported_module.graph)
 
         self.assertTrue(torch.allclose(exported_module(inps), model_to_trace(inps)))
```

Comment:
Can we add some tests for different types of inputs? ex. dataclasses, lists, dicts, kwargs